src/slashCommand/commands/ping.command.ts

```diff
@@ -1,7 +1,17 @@
-import { SlashCommandBuilder } from 'discord.js';
+import {
+  CacheType,
+  ChatInputCommandInteraction,
+  SlashCommandBuilder,
+} from 'discord.js';
 
 export default class PingCommand {
   public static slashCommandBuilder = new SlashCommandBuilder()
     .setName('ping')
     .setDescription('Replies with pong!');
+
+  public static async handleCommand(
+    interaction: ChatInputCommandInteraction<CacheType>,
+  ): Promise<void> {
+    await interaction.reply('Pong!');
+  }
```

Comment:
Likely that we want to instantiate this class at some point but this is a really good start for now.

Reply:
I was looking through the docs for some reason and found this way of using the SlashCommandBuilder to define a reaction to a command. Have not tried it, but we could keep it in mind for later.
```javascript
module.exports = {
	data: new SlashCommandBuilder()
		.setName('ping')
		.setDescription('Replies with Pong!'),
	async execute(interaction) {
		await interaction.reply('Pong!');
	},
};```